plugins/devtools/README.md

```diff
@@ -0,0 +1,375 @@
+# DevTools
+
+Welcome to the DevTools plugin!
+
+## Features
+
+The DevTools plugin comes with two tabs out of the box.
+
+### Info
+
+Lists helpful information about your current running Backstage instance such as: OS, NodeJS version, Backstage version, and package versions.
+
+![Example of Info tab](./docs/devtools-info-tab.png)
+
+### Config
+
+Lists the configuration being used by your current running Backstage instance.
+
+![Example of Config tab](./docs/devtools-config-tab.png)
+
+## Optional Features
+
+The DevTools plugin can be setup with other tabs with additional helpful features.
+
+### External Dependencies
+
+Lists the status of configured External Dependencies based on your current running Backstage instance's ability to reach them
```

Comment:
```suggestion
Lists the status of configured External Dependencies based on your current running Backstage instance's ability to reach them.
```

nit 😆

Reply:
Yeah, this is a bad habit of mine. Fixed!